crypto/jwx/jws_dilithium.go

```diff
@@ -0,0 +1,86 @@
+package jwx
+
+import (
+	"fmt"
+
+	"github.com/cloudflare/circl/sign/dilithium"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jws"
+)
+
+func init() {
+	jws.RegisterSigner(DilithiumMode2Alg, jws.SignerFactoryFn(NewDilithiumMode2Signer))
+	jws.RegisterVerifier(DilithiumMode2Alg, jws.VerifierFactoryFn(NewDilithiumMode2Verifier))
+	jws.RegisterSigner(DilithiumMode3Alg, jws.SignerFactoryFn(NewDilithiumMode3Signer))
+	jws.RegisterVerifier(DilithiumMode3Alg, jws.VerifierFactoryFn(NewDilithiumMode3Verifier))
+	jws.RegisterSigner(DilithiumMode5Alg, jws.SignerFactoryFn(NewDilithiumMode5Signer))
+	jws.RegisterVerifier(DilithiumMode5Alg, jws.VerifierFactoryFn(NewDilithiumMode5Verifier))
+}
```

Comment:
Users of the SDK may not want to accept PQC yet. Thus, I recommend that this registration should be up to the user of the library, instead of an `init` function.

Reply:
overhead is small. they may not want any key type, should they have to enable them individually?

this is an opinionated lib that provides broad support. my take is keep it as is.